homeassistant/components/blink/diagnostics.py

```diff
@@ -0,0 +1,35 @@
+"""Diagnostics support for Aladdin Connect."""
+from __future__ import annotations
+
+from typing import Any
+
+from blinkpy.blinkpy import Blink
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+TO_REDACT = {"serial", "macaddress", "username", "password", "token"}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+
+    api: Blink = hass.data[DOMAIN][config_entry.entry_id].api
+
+    cameras = {api.cameras[camera] for camera in api.cameras}
+    data = {}
+    for camera in cameras:
+        data[camera.name] = dict(camera.attributes.items())
```

Comment:
```suggestion
    data = {}
    for idx, camera in api.cameras:
        data[camera.name] = dict(camera.attributes.items())
```
Isn't this like the same? I don't see why we're iterating over the same list twice

Reply:
Not really, it's getting `api.cameras[camera]` into the cameras list, then putting the attributes for each camera by name into a "nice" data dictionary. Maybe I could get at it with `api.cameras[camera].attributes.items()` rather than `api.cameras` but I'm not sure how to get the `[camera]` in the second loop. 
The proposed code blows up with "too many values to unpack"

Reply:
Ok - needed `api.cameras.items()` - Python 101.